src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.bigForm {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.searchLabel {
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+  top: 0;
+  left: 26px;
+}
+
+.bigInput {
+  font-family: "Avenir", sans-serif;
+  padding-left: 62px;
+  outline: none;
+  height: 70px;
+  width: 100%;
+  font-weight: bold;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  color: #3d4e61;
+  background-color: #fff;
+}
+
+.bigInput::placeholder {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  text-shadow: none;
+  color: #3d4e61;
+}
+
+.bigInput:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigInput:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.icon--small {
+  top: 5px;
+  left: 13px;
+}
+
+.smallInput {
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  padding-left: 33px;
+  background-color: #fff;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 19px;
```

Comment:
It's better to use a percentage or multiplier value for line height.
`line-height: 1.4`, `line-height: 150%`, etc.